xapian-bindings/python3/docs/examples/xapian-rank.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+#
+# Simple example script demonstrating how to re-rank using the trained model.
+#
+# Copyright (C) 2019 Vaibhav Kansagara
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+# USA
+
+import sys
+import xapian
+import xapianletor
+
+# We require at least three command line arguments.
+if len(sys.argv) < 4:
+    print("Usage: PATH_TO_DATABASE MSIZE QUERY",
```

Comment:
Again allows but ignores extra arguments, and the program name is missing.  Please fix any further instances too.

Reply:
fixed in a fixup commit.